CHANGELOG.md

```diff
@@ -1,19 +1,62 @@
-# 2.1.0 (2017-01-XX)
+# 2.1.0 (2017-02-TBD)
+### Version highlights:
+1. Added full support to OneView Rest API version 300 for the hardware variants C7000 and Synergy to the already existing features.
+2. Implemented a Oneview_resource class for the resources to inherit from, reducing code duplication and complexity.
+3. Enabled the 'puppet resource <oneview_type>' for most of the providers, allowing the resources to be queried.
+4. Overall refactor and code cleanup.
 
 ### Notes
-This release extends the full support for the Synergy and C7000 APIs to all the resources previously supported in version 1.0.0, and a few new resources specific to API300.
+This release extends the full support for the Synergy and C7000 APIs to all the resources previously supported, and adds a few new resources specific to Synergy.
 
-### Features supported
-- Ethernet networks
-- FC networks
-- FCOE networks
-- Network sets
-- Fabrics
-- Connection templates
-- Datacenters
-- Enclosures
-- Enclosure Groups
+### Puppet Types Added
+- Oneview_drive_enclosure
+- Oneview_sas_interconnect
+- Oneview_sas_logical_interconnect
+- Oneview_sas_logical_interconnect_group
 
+### Oneview Features supported
+- Connection template
+- Datacenter
+- Enclosure
+- Ethernet network
+- Fabrics
+- FC network
+- FCoE network
+- Firmware bundles
+- Firmware drivers
+- Interconnect
+- Logical downlink
+- Logical enclosure
+- Logical interconnect
+- Logical interconnect Group
+- Uplink Set
+- Logical switch
+- Logical switch group
+- Managed SANs
+- Network set
+- Power devices
+- Racks
+- SAN managers
+- Server hardware
+- Server hardware type
+- Server profile
+- Server profile template
+- Storage pools
+- Storage systems
+- Switches
+- Volume
+- Volume attachment
+- Volume template
+- Drive Enclosures
```

Comment:
Since this list have a lot of resources, I would prefer the items sorted by name.